### PR TITLE
Mricrogl

### PR DESCRIPTION
**Pull request by TheChymera**
New neuroimaging software package.

### CODE REVIEW

File: sci-biology/mricrogl/mricrogl-9999.ebuild [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+	insinto /usr/bin/shaders
+	doins shaders/*.txt
+
+	doicon mricrogl.ico
+	make_desktop_entry MRIcroGL MRIcroGL mricrogl
+$
+}
+
+pkg_postinst() {
+	gnome2_icon_cache_update
```
Comment (SoapGentoo) [inline sci-biology/mricrogl/mricrogl-9999.ebuild]: how can you do this without inheriting `gnome2-utils.eclass` ?

File: sci-biology/mricrogl/mricrogl-9999.ebuild [hunk 2]
```diff
@@ -0,0 +1,41 @@
+# Copyright 1999-2015 Gentoo Foundation
```
Comment (SoapGentoo) [inline sci-biology/mricrogl/mricrogl-9999.ebuild]: Year pls.

File: sci-biology/mricrogl/mricrogl-9999.ebuild [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI="5"
```
Comment (SoapGentoo) [inline sci-biology/mricrogl/mricrogl-9999.ebuild]: why no `EAPI=6`?

File: sci-biology/mricrogl/mricrogl-9999.ebuild [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+
+src_install() {
+	dobin MRIcroGL
+
+	insinto /usr/bin/shaders
+	doins shaders/*.txt
+
+	doicon mricrogl.ico
+	make_desktop_entry MRIcroGL MRIcroGL mricrogl
+$
```
Comment (SoapGentoo) [inline sci-biology/mricrogl/mricrogl-9999.ebuild]: ???

File: sci-biology/mricrogl/mricrogl-9999.ebuild (hunk 1 above)
Comment (TheChymera) [inline sci-biology/mricrogl/mricrogl-9999.ebuild]: I tried to emulate [this](https://github.com/gentoo-science/sci/blob/master/sci-biology/mricron/mricron-9999.ebuild) - portage has not complained.

Comment (jlec) [inline sci-biology/mricrogl/mricrogl-9999.ebuild]: Same there. You are missing the eclass.

**Comment by TheChymera** [comment]
@jlec ?

**Pull request merged**